etcdmgmt/etcd-mgmt.go

```diff
@@ -35,6 +36,11 @@ func StartEtcd() error {
 		"-initial-advertise-peer-urls", initialAdvPeerUrls,
 		"--initial-cluster", "default="+listenPeerUrls)
 
+	// Don't kill chlid process (etcd) upon ^C (SIGINT) of main glusterd process
```

Comment:
@atinmu 
>> How about SIGKILL, SIGTERM? Aren't you aware that these signals are for shutting down a process?

I am aware very well about these all the signal. It seems to miscommunication in my last comment. I asked that how are you getting child process killed without ^C. 
suppose my glusterd2 is running and glusterd2 have started etcd child process and if you issue kill (SIGKILL/SIGTERM) from terminal then its not killing child process (etcd daemon). It will only kill etcd daemon when you issue ^C from terminal. typing ^C in unix send signal to every process group. I asked how did you killed etcd daemon from SIGKILL and SIGTERM.

>> Regarding running etcd instance from the code its straight forward that there is no check of whether a running etcd instance is available and if so etcd start shoild be skipped. When I pulled your changes I could easily see a defunct etcd process on a glusterd restart. I am not sure about what you have tested.

you should have described this in the bug itself. doing this is not a big deal.

 >> Moreover, this change will solve the issue of not letting the etcd instance go down on a glusterd restart but this doesn't cover the definition of daemonizing a process which should be a framework to manage multiple daemons.


While creating issue you did not talked about its framework so i was not aware about it. i was mainly focusing on "etcd instance can't be brought down when glusterd goes down since the other consumers like NSR would still be able to access it" part as you have created in issue #66